tests/test_functional.py

```diff
@@ -560,5 +560,19 @@ def test_obal_changelog_custom():
 
 @obal_cli_test(repotype='upstream')
 def test_obal_srpm():
-    assert_obal_success(['srpm', 'hello'])
-    assert_mockbin_log(['tito build --srpm --offline --output {pwd}/SRPMs'])
+    assert_obal_success(['srpm', 'hello', 'foo'])
+    assert_mockbin_log([
+        'tito build --srpm --offline --output {pwd}/SRPMs',
+        'tito build --srpm --offline --output {pwd}/SRPMs'
+    ])
+
+
+@obal_cli_test(repotype='upstream')
+def test_obal_mock():
+    assert_obal_success(['mock', 'hello', 'foo', '--config', 'mock/el7.cfg'])
```

Comment:
`--config` is just pointing at the default, can be dropped

Reply:
Optional but common to use. Should I write a separate test all together for it?

Reply:
we seldomly test optional params unless they heavily influence the execution logic. so if *I'd* write that test, I'd just write one without `--config` and pretend `--config` works if the main test succeeds. but it also doesn't hurt the way it is now.